docs/add-new-route-testcase.md

```diff
@@ -0,0 +1,31 @@
+# Add New Route testcase
+
+ ## Prerequisites
+
+User has acount in the system already created by system admin.
+User is log in. 
+User is on a main view.
+
+ ## Steps
+
+ ### Step 1
+User clicks "Add New Route" button at the top of the page.
+If User use screen with lower resolution "Add New Route" button is hidden in hamburger menu.
```

Comment:
We should think of separate scenarios for mobile views. It would be more clean in my opinion. As we would not strict breakpoints (or I am not sure exactly, how Vue will handle that), I would go with separate scenarios for every view with RWD tag (just to ensure, that the test / happy path is performed on mobile device).